File: fix_encoding.py
import re

# Read the file
with open(r'c:\fiverr projects\hse new client\new\hse-hub-main\src\pages\RiskAssessments.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace HTML entities with proper characters
content = content.replace('&amp;gt;', '>')
content = content.replace('&gt;', '>')
content = content.replace('&amp;lt;', '<')
content = content.replace('&lt;', '<')

# Write back
with open(r'c:\fiverr projects\hse new client\new\hse-hub-main\src\pages\RiskAssessments.tsx', 'w', encoding='utf-8') as f:
    f.write(content)

print("Fixed HTML entities in RiskAssessments.tsx")

File: fix-encoding.js
const fs = require('fs');

const filePath = 'c:\\fiverr projects\\hse new client\\new\\hse-hub-main\\src\\pages\\RiskAssessments.tsx';

// Read file
let content = fs.readFileSync(filePath, 'utf8');

// Fix HTML entities
content = content.replace(/&amp;gt;/g, '>');
content = content.replace(/&gt;/g, '>');
content = content.replace(/&amp;lt;/g, '<');
content = content.replace(/&lt;/g, '<');

// Write back
fs.writeFileSync(filePath, content, 'utf8');

console.log('Fixed HTML entities!');
